apps/coscrad-frontend-e2e/src/e2e/resources/audio-items/audio-annotation-flow.e2e.cy.ts

```diff
@@ -0,0 +1,218 @@
+import { AggregateType, MIMEType, ResourceType } from '@coscrad/api-interfaces';
+import { buildDummyAggregateCompositeIdentifier } from '../../../support/utilities';
+
+const audioTitleInLanguage = 'Audio Title (Lang)';
+
+const basicAudioAggregateCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.audioItem,
+    1
+);
+
+const mediaItemCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.mediaItem,
+    21
+);
+
+const { id: basicAudioId } = basicAudioAggregateCompositeIdentifier;
+
+const audioDetailRoute = `/Resources/AudioItems/${basicAudioId}`;
+
+const validUrl =
+    'https://coscrad.org/wp-content/uploads/2023/05/metal-mondays-mock2_370934__karolist__guitar-solo.mp3';
+
+const buildDetailRoute = (audioItemId: string) => `/Resources/AudioItems/${audioItemId}`;
+
+const millisecondsToRoundedSeconds = (milliseconds: number) => {
+    const seconds = milliseconds / 1000;
+
+    // round to one decimal place
+    return Math.round(seconds * 10) / 10;
+};
+
+const isWithinToleranceOfFloat = (value: number, targetValue: number) => {
```

Comment:
The tolerance should be a third, optional parameter that is set to a reasonable (not 0.5!) default.

Reply:
Using .closeTo()